tests/unit_tests/test_color_classifier.py

```diff
@@ -0,0 +1,115 @@
+from UpdatedImageProcessing import ColorClassifier
+import unittest
+import os
+
+class ColorClassifierTestCase(unittest.TestCase):
+
+	def test_get_color(self):
+		cc = ColorClassifier(os.path.realpath(__file__)[0:-24] + "test_color_classifier.png")
```

Comment:
This path should be relative based on execution from base directory of SUAS repository.

Reply:
@jvillemare This is the only outstanding comment left.

Reply:
I know—sorry. I addressed 2 of your 3 comments because they were easier to fix than the relative path.

I don't know what was going wrong when I was first writing the path to the test image, but just giving the filename works in commit https://github.com/FlintHill/SUAS-Competition/pull/108/commits/0226b6a79db18105c8ff76259412d5d85ae856cc & https://github.com/FlintHill/SUAS-Competition/pull/108/commits/2459cebaa5b6353073cd21214886ec495814c0d8